pkg/process/checks/process.go

```diff
@@ -266,6 +278,15 @@ func (p *ProcessCheck) run(groupID int32, collectRealTime bool) (RunResult, erro
 	return result, nil
 }
 
+func (p *ProcessCheck) generateHints() int32 {
+	var hints int32
+
+	if p.checkCount%p.skipAmount == 0 {
```

Comment:
Looks like this would crash if the config value was explicitly set to 0, not sure how defensive we want to be about that